mobilebot/mobilebot.c

```diff
@@ -67,6 +67,7 @@ int main(){
 	mb_initialize_controller();
```

Comment:
Do we also need to add `mb_destory_controller()` down the line during exit?

Reply:
Adding the controller cleanup to the end